### PR TITLE
LP-150 Add localisation for web pages

### PR DESCRIPTION
**Pull request by markpit**
### JIRA link
https://companieshouse.atlassian.net/browse/LP-150

### Change description
Adding the ch-node-utils library to bring in the localisation functionality
Added transalation.json files for en and cy under /locales
Updated the build commands to copy the locales folder into /dist
Added pre-push hook to run tests before pushing to github

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes look good on mobile
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/test/localisation.test.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  test("renders the start page with English text when an unsupported language is given", async () => {
+    setLocalesEnabled(true);
+
+    const resp = await request(app).get(config.START_URL + "?lang=pp");
+
+    expect(resp.text).toContain(enStartPageText.startTitle);
+  });
+
+  test("renders the start page with English text when an unsupported language is given", async () => {
```
Comment (lduranteau) [inline src/test/localisation.test.ts:43]: duplicate test

File: src/test/localisation.test.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+    expect(resp.text).toContain(enStartPageText.startTitle);
+  });
+
+  test("renders the start page with the language nav bar when localisation is switched on", async () => {
+    setLocalesEnabled(true);
+
+    const resp = await request(app).get(config.START_URL);
+
+    expect(resp.text).toContain("English");
```
Comment (lduranteau) [inline src/test/localisation.test.ts:72]: these two expect can be moved to line 57

File: src/test/localisation.test.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    expect(resp.text).toContain("Cymraeg");
+  });
+
+  test("renders the start page without the language nav bar when localisation is switched off", async () => {
+    setLocalesEnabled(false);
+
+    const resp = await request(app).get(config.START_URL);
+
+    expect(resp.text).not.toContain("English");
```
Comment (lduranteau) [inline src/test/localisation.test.ts:81]: these two expect can be moved to line 64

**Review by lduranteau — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**